interop/src/clients/corecrypto/ffi.rs

```diff
@@ -29,29 +30,38 @@ use crate::{
 pub(crate) struct CoreCryptoFfiClient {
     cc: CoreCrypto,
     client_id: Vec<u8>,
+    temp_file: NamedTempFile,
```

Comment:
Maybe also add an `#[allow(unused)]` here, along with a similar note to the one where this is initialized. 

Alternately, just naming this member `_temp_file` silences the warnings, but also preserves the lifetime so it doesn't drop until the struct does.

Reply:
> Alternately, just naming this member _temp_file silences the warnings, but also preserves the lifetime so it doesn't drop until the struct does.

Ah, nice, I'm going to use that, thanks!